### PR TITLE
29 rating edition

### PR DESCRIPTION
**Pull request by MaximePikss**

### CODE REVIEW

**Review by RoxaneDrelon — DISMISSED** [review]
SnE, waiting for Loïc to see the "no-console: false"

File: src/components/capsulelist/CapsuleList.jsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+//             color="secondary"
+//             className={classes.root}
+//             aria-label="change"
+//           >
+//             <MapIcon color="primary" />
+//           </Fab>
+//         </div>
+//       )}
+//     </>
+//   );
```
Comment (LoicBrassart) [inline src/components/capsulelist/CapsuleList.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some dead code

**Review by RoxaneDrelon — CHANGES_REQUESTED** [review]
ESLint, problème Capsule/CapsuleList

**Review by RoxaneDrelon — APPROVED** [review]
SnE, "test" is normal

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**